org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/StandardProjectsManager.java

```diff
@@ -193,7 +194,12 @@ public void fileChanged(String uriString, CHANGE_TYPE changeType) {
 			List<URI> uris = getURIs(formatterUrl);
 			boolean changed = false;
 			for (URI formatterUri : uris) {
-				if (formatterUri.equals(uri)) {
+				if (Platform.OS_WIN32.equals(Platform.getOS()) && URIUtil.isFileURI(formatterUri) && URIUtil.isFileURI(uri)) {
```

Comment:
Could you add similar code for the settingsUri variable - https://github.com/eclipse/eclipse.jdt.ls/pull/1738/commits/008b542479ec26991c0835053031318a02dead19#diff-f23d0258ce24117c110acd6753e8490a09fbdf0bc039d662bdffe279c8c28469L177 ?

Reply:
added in https://github.com/eclipse/eclipse.jdt.ls/pull/1738/commits/c88f8f58c051c63af12a99ecd997cc8ad69ff347